### PR TITLE
Language control

### PR DESCRIPTION
**Pull request by atomoc**

### CODE REVIEW

File: templates/categories.tpl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <ul class="categories" itemscope itemtype="http://www.schema.org/ItemList">
-	<p>Categories</p>
+	<p class="cat-title">[[category:cat_title:Categories]]</p>
```
Comment (julianlam) [inline templates/categories.tpl]: Should be written `[[header.categories]]`, possible. I'd like @barisusakli's comments on where the exact language key should be placed

File: templates/partials/categories/lastpost.tpl [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 	<!-- IF !posts.length -->
 	<div class="post-content">
-		No new posts.
+		<div class="no_posts">[[category:no_posts:No new posts.]]</div>
```
Comment (julianlam) [inline templates/partials/categories/lastpost.tpl]: `<div class="no_posts">[[category:no_posts]]</div>`. The english translation should be saved in `public/language/en_GB/category.json` under the key `no_posts`.

File: templates/partials/category/subcategory.tpl [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <div class="subcategory">
-	<!-- IF children.length --><p>Subcategories</p><!-- ENDIF children.length -->
+	<!-- IF children.length --><p class="subcattitle">[[category:subcat_title:Subcategories]]</p><!-- ENDIF children.length -->
```
Comment (julianlam) [inline templates/partials/category/subcategory.tpl]: Maybe `[[header:subcategories]]`, if @barisusakli signs off on that

File: templates/partials/category/subcategory.tpl [hunk 4]
```diff
@@ -50,7 +50,7 @@
 
 					<!-- IF !posts.length -->
 					<div class="post-content">
-						No new posts.
+						<div class="no_posts">[[category:no_posts:No new posts.]]</div>
```
Comment (julianlam) [inline templates/partials/category/subcategory.tpl]: Same as above, should be `<div class="no_posts">[[category:no_posts]]</div>`

**Comment by atomoc** [comment]
Ready

**Comment by julianlam** [comment]
Hi @atomoc, thanks for the changes. Why have classes been added to the changed lines?

**Comment by atomoc** [comment]
For CSS

**Comment by atomoc** [comment]
Remove?

**Comment by julianlam** [comment]
I believe so -- unless they're used in the persona theme CSS itself, otherwise we would not know where it came from, and it may get removed

**Comment by julianlam** [comment]
Looks like this PR has no more lines changed :smile:

**Comment by atomoc** [comment]
:)

**Pull request closed without merging**